### DISCUSSION TITLE
Please ignore reply messages

### DISCUSSION THREAD

**Discussion by k0ste** [metalmatze/alertmanager-bot#174] [Help/Support]
![Screen Shot 2021-03-02 at 15 35 18](https://user-images.githubusercontent.com/7759548/109651175-38d4f980-7b6f-11eb-8ee5-22d184f010da.png)
![Screen Shot 2021-03-02 at 15 35 34](https://user-images.githubusercontent.com/7759548/109651181-3a062680-7b6f-11eb-8613-a7d1901173b0.png)

Alert -> reply to alert for CC' `@man_who_fix_the_alert` -> `@man_who_fix_the_alert` reply to me for my reply...

Bot should not react on this messages

**Comment by metalmatze** [comment]
I think that should be fixed and getting a lot better in a next release with the latest version of telebot! :)

**Reply by k0ste** [reply]
I was updated bot to last git version (build from source), now alertmanager get's 404:

```
Mar 02 20:24:24 mon alertmanager[6545]: level=error ts=2021-03-02T13:24:24.246Z caller=dispatch.go:309 component=dispatcher msg="Notify for alerts failed" num_alerts=3 err="alertmananger-bot/webhook[0]: notify retry canceled due to unrecoverable error after 1 attempts: unexpected status code 404: http://127.0.0.1:8085"
```

On alertmanager-bot side no new logs. All settings was the same. What I miss?

**Reply by k0ste** [reply]
Oh, I think bot miss subscribed chats, but I think is something new...

![image](https://user-images.githubusercontent.com/7759548/109656031-03330f00-7b75-11eb-958b-b1d305765ba9.png)

Can you, with development make changes to CHANGELOG.md, because currently we don't know how to migrate to new version of bot.

P.S.: how-to solve current issues? 🙄

**Reply by metalmatze** [reply]
You need to change the URLs in the webhook in the alertmanager configuration.
These changes haven't been released (thus no release was cut yet) and therefore I didn't write the CHANGELOG.md changes yet.
I'm glad you're trying things as the come hot out of the oven.

Each chat (private) and group now need to have their own webhook receiver in the alertmanager config specifying the URL that the bot gave you in the message seen in the screenshot.

**Reply by k0ste** [reply]
I'm exactly mean, when PR merged - CHANGELOG.MD should be updated, should if possible 🤫

**Reply by metalmatze** [reply]
Yes, should be possible. On the other hand this a personal hobby project and I don't have that high standards for what I do in my free time. :upside_down_face: 
So until I cut a release and write a bunch of sentences I think you're on your own. Still, I'm happy to help here if I can.

**Reply by k0ste** [reply]
@metalmatze, I still experiment with master branch

```
Mar 10 20:50:02 mon alertmanager-bot[22552]: level=debug ts=2021-03-10T13:50:02.160783045Z caller=bot.go:318 component=telegram msg="message received" text=/status
Mar 10 20:50:02 mon alertmanager-bot[22552]: level=warn ts=2021-03-10T13:50:02.1610766Z caller=bot.go:408 component=telegram msg="failed to get status" err="unknown error (status 404): {resp:0xc0000b6c60} "
```

What exactly query is executed?

**Reply by k0ste** [reply]
Found what is it - custom alertmanager `web.route-prefix`.

**Reply by metalmatze** [reply]
Yep, the bot moved to using a generated alertmanager api v2 client.
https://github.com/prometheus/alertmanager/tree/master/api/v2
What was failing is this the call to get the status at `/api/v2/status` so it only makes sense with `web.route-prefix` :+1: 
See 
https://github.com/prometheus/alertmanager/blob/master/api/v2/client/general/general_client.go#L55
